lambda-function-connect-rds-iam/example.cs

```diff
@@ -0,0 +1,73 @@
+using Amazon.Lambda.Core;
+using Amazon.RDS.Util;
+using MySql.Data.MySqlClient;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace aws_rds;
+
+public class Function
+{
+
+    /// <summary>
+    /// A simple function that takes a string and does a ToUpper
+    /// </summary>
+    /// <param name="input">The event for the Lambda function handler to process.</param>
+    /// <param name="context">The ILambdaContext that provides methods for logging and describing the Lambda environment.</param>
+    /// <returns></returns>
+    public string FunctionHandler(string input, ILambdaContext context)
+    {
+        /// Obtain authentication token
+        var authToken = RDSAuthTokenGenerator.GenerateAuthToken(
+            Environment.GetEnvironmentVariable("RDS_ENDPOINT"),
+            Convert.ToInt32(Environment.GetEnvironmentVariable("RDS_PORT")),
+            Environment.GetEnvironmentVariable("RDS_USERNAME")
+        );
+
+        /// Build the Connection String with the Token 
+        string connectionString = $"Server={Environment.GetEnvironmentVariable("RDS_ENDPOINT")};" +
+                                  $"Port={Environment.GetEnvironmentVariable("RDS_PORT")};" +
+                                  $"Uid={Environment.GetEnvironmentVariable("RDS_USERNAME")};" +
+                                  $"Pwd={authToken};";
+
+
+        try
+        {
+            /// Connect to the DB
+            using (MySqlConnection connection = new MySqlConnection(connectionString))
+            {
+                connection.Open();
+                Console.WriteLine("Successfully connected to the database.");
+
+                string sql = "SELECT @param1 + @param2 AS Sum";
+
+                //Build the Command
+                using (MySqlCommand cmd = new MySqlCommand(sql, connection))
+                {
+                    cmd.Parameters.AddWithValue("@param1", 1);
+                    cmd.Parameters.AddWithValue("@param2", 2);
+
+                    // Execute the Command
+                    using (MySqlDataReader reader = cmd.ExecuteReader())
+                    {
+                        if (reader.Read())
+                        {
+                            int sum = reader.GetInt32("Sum");
+                            return $"The sum is: {sum}";
+                        }
+                    }
+                }
+
+                connection.Close();
+                Console.WriteLine("Connection closed.");
+            }
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine($"Error: {ex.Message}");
+        }
+
+        return "Failed to connect to the database.";
```

Comment:
Can we return something else here rather than this string? 

Maybe a dummy result (as in Python) https://github.com/aws-samples/serverless-snippets/blob/main/lambda-function-connect-rds-iam/example.py
or response (as in Java) https://github.com/aws-samples/serverless-snippets/blob/main/lambda-function-connect-rds-iam/example.java